tests/cypress/integration/plugins/morph.spec.js

```diff
@@ -255,3 +255,34 @@ test('can morph text nodes',
         get('h2').should(haveHtml('Foo <br> Baz'))
     },
 )
+
+test.only('can morph with added element before and siblings are different',
```

Comment:
@calebporzio This is probably not intentional _(referring to the `.only`)_, right ? ^^